service/app/controllers/usermanagement/UserMergeController.java

```diff
@@ -2,20 +2,25 @@
 
 import controllers.BaseController;
 import org.sunbird.common.models.util.ActorOperations;
+import org.sunbird.common.models.util.JsonKey;
 import org.sunbird.common.request.Request;
 import org.sunbird.common.request.UserRequestValidator;
 import play.libs.F.Promise;
 import play.mvc.Result;
 
+import java.util.Map;
+
 public class UserMergeController extends BaseController {
 
     public Promise<Result> mergeUser() {
+      String authUserToken = ctx().request().getHeader(JsonKey.X_AUTHENTICATED_USER_TOKEN);
+      String sourceUserToken = ctx().request().getHeader(JsonKey.X_SOURCE_USER_TOKEN);
         return handleRequest(ActorOperations.MERGE_USER.getValue(),
                 request().body().asJson(),
                 req -> {
                     Request request = (Request) req;
-                    new UserRequestValidator().validateUserMergeRequest(request);
+                    new UserRequestValidator().validateUserMergeRequest(request, authUserToken, sourceUserToken);
                     return null;
-                });
+                },(Map)request().headers());
```

Comment:
we already received header values , so do we still need to pass (Map)request().headers()

Reply:
I seen this is the way to pass Headers to the request, else it won't get header values in actor.